### PR TITLE
Tolerate any taints in ovnkube-master daemonset

### PR DESCRIPTION
**Pull request by mangelajo**
Closes: https://issues.redhat.com/browse/USHIFT-237

This is a rebased version of Pablo's work here: https://github.com/openshift/microshift/pull/818

**Which issue(s) this PR addresses**:

Closes #<Issue Number>

### CODE REVIEW

**Comment by zshi-redhat** [comment]
test e2e-openshift-conformance-sig-node

File: assets/components/ovn/master/daemonset.yaml [hunk 1]
```diff
@@ -482,11 +482,4 @@ spec:
...
           optional: true
       tolerations:
-      - key: "node-role.kubernetes.io/master"
-        operator: "Exists"
-      - key: "node.kubernetes.io/not-ready"
-        operator: "Exists"
-      - key: "node.kubernetes.io/unreachable"
-        operator: "Exists"
-      - key: "node.kubernetes.io/network-unavailable"
         operator: "Exists"
```
Comment (mangelajo) [inline assets/components/ovn/master/daemonset.yaml:485]: I'm missing the "-" in front of operator

**Comment by mangelajo** [comment]
It may work now.

**Pull request merged**